### PR TITLE
refactor: typo fixes

### PR DESCRIPTION
**Pull request by ayusharma**
**Type:** Refactor 

The following has been addressed in the PR:

*  There is a related issue?
*  Unit or Functional tests are included in the PR

**Description:**
- small code changes

### CODE REVIEW

File: src/lib/config-utils.js [hunk 1]
```diff
@@ -21,27 +21,26 @@ const BLACKLIST = {
...
+    if (value == null) {
       continue;
     }
-
     // if it's a string, split it to array
-    if (_.isString(arguments[i])) {
-      result.push(arguments[i].split(/\s+/));
-    } else if (Array.isArray(arguments[i])) {
-      result.push(arguments[i]);
+    if (typeof value === 'string') {
```
Comment (juanpicado) [inline src/lib/config-utils.js]: I'd create an utility for use in several places `typeof value === 'string'`

File: src/lib/config-utils.js [hunk 2]
```diff
@@ -21,27 +21,26 @@ const BLACKLIST = {
 };
 
 /**
- * Normalise user list.
+ * Normalize user list.
  * @return {Array}
  */
-export function normalizeUserlist(oldFormat: any, newFormat: any) {
+export function normalizeUserList(...args: any): Array<string> {
```
Comment (juanpicado) [inline src/lib/config-utils.js]: `normalizeUserList` to be sure this works fine. A unit test is required.  I'd do it undoing this changes and test both implementations.

File: src/lib/status-cats.js [hunk 3]
```diff
@@ -53,16 +53,15 @@ const images = {
 };
 
 module.exports.get_image = function(status) {
-  if (status in images) {
-    return 'http://flic.kr/p/' + images[status];
-    // return 'https://secure.flickr.com/photos/girliemac/'+images[status]+'/in/set-72157628409467125/lightbox/'
+  if (Object.prototype.hasOwnProperty.call(images, status)) {
```
Comment (juanpicado) [inline src/lib/status-cats.js]: Get rid of this file, it's useless.

File: src/lib/config-utils.js (hunk 1 above)
Comment (ayusharma) [reply src/lib/config-utils.js]: where ?

File: src/lib/status-cats.js (hunk 3 above)
Comment (ayusharma) [reply src/lib/status-cats.js]: Done :)

File: src/lib/local-storage.js [hunk 4]
```diff
@@ -56,7 +43,7 @@ class LocalStorage implements IStorage {
     }
 
     storage.createPackage(name, generatePackageTemplate(name), err => {
-      if (_.isNull(err) === false && err.code === fileExist) {
+      if (_.isNull(err) === false && err.code === STORAGE.FILE_EXIST) {
```
Comment (juanpicado) [inline src/lib/local-storage.js]: `STORAGE.FILE_EXIST` I'd rather call the namespace something as `XXX_ERROR`. The idea is should be used by plugins and some devs might find it misleading.

File: src/lib/local-storage.js [hunk 5]
```diff
@@ -417,7 +404,7 @@ class LocalStorage implements IStorage {
       _transform.apply(uploadStream, arguments);
     };
 
-    if (name === pkgFileName || name === '__proto__') {
+    if (name === STORAGE.PACKAGE_FILE_NAME || name === '__proto__') {
```
Comment (juanpicado) [inline src/lib/local-storage.js]: In this case, makes total sense.

File: src/lib/local-storage.js (hunk 4 above)
Comment (juanpicado) [reply src/lib/local-storage.js]: The same for `STORAGE.NO_SUCH_FILE`

File: src/lib/search.js [hunk 6]
```diff
@@ -70,12 +70,12 @@ class Search implements IWebSearch {
...
   /**
-   * Force a reindex.
+   * Force a re-index.
    */
   reindex() {
     let self = this;
-    this.storage.getLocalDatabase(function(err, packages) {
-      if (err) throw err; // that function shouldn't produce any
+    this.storage.getLocalDatabase(function(error, packages) {
+      if (error) throw error; // that function shouldn't produce any
```
Comment (juanpicado) [inline src/lib/search.js]: Curly brackets here `{ (breakLine) xx  (breakLine)}`

**Pull request merged**